model/system_metrics.go

```diff
@@ -183,7 +186,7 @@ func (sm *SystemMetrics) Append(ctx context.Context, data []byte) error {
 		return nil
 	}
 
-	key := fmt.Sprint(utility.UnixMilli(time.Now()))
+	key := metricType + fmt.Sprint(utility.UnixMilli(time.Now()))
```

Comment:
can you make this `fmt.Sprintf("%s-%v", metricType, utility.UnixMilli(time.Now()))`?